File: coinsaver-front/src/app/dtos/transactions/response/performance.response.dto.ts
import { PerformanceDivisionsResponseDto } from "./performance-divisions.response.dto";
import { PerformanceEconomyResponseDto } from "./performance-economy.response.dto";
import { PerformanceMonthlyBalanceResponseDto } from "./performance-monthly-balance.response.dto";

export class PerformanceResponseDto {
  constructor(
    public economy: PerformanceEconomyResponseDto,
    public monthlyBalance: PerformanceMonthlyBalanceResponseDto,
    public division: PerformanceDivisionsResponseDto,
  ) {}
}

File: coinsaver-api/src/main/java/com/coinsaver/api/dtos/response/PerformanceResponseDto.java
package com.coinsaver.api.dtos.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PerformanceResponseDto {

    private PerformanceEconomyResponseDto economy;
    private PerformanceMonthlyBalanceResponseDto monthlyBalance;
    private PerformanceDivisionsResponseDto division;

}
